tests/Exception/UnknownFileTypeExceptionTest.php

```diff
@@ -7,11 +7,27 @@
 
 final class UnknownFileTypeExceptionTest extends PHPUnit_Framework_TestCase
 {
-    public function testItIsAnInstanceOfThePackagesRuntimeException()
+    public function testItImplementsTheBaseExceptionType()
     {
         $this->assertInstanceOf(
-            'TomPHP\ConfigServiceProvider\Exception\RuntimeException',
+            'TomPHP\ConfigServiceProvider\Exception\Exception',
             new UnknownFileTypeException()
         );
     }
+
+    public function testItIsADomainException()
+    {
+        $this->assertInstanceOf('DomainException', new UnknownFileTypeException());
+    }
+
+    public function testItCanBeCreatedFromFileExtension()
+    {
+        $this->assertEquals(
+            'No reader configured for ".yml" files; readers are available for [".json", ".php"].',
+            UnknownFileTypeException::fromFileExtension(
+                '.yml',
+                ['.json', '.php']
+            )->getMessage()
+        );
```

Comment:
@localheinz agreed. I guess I've just got out of the habit as I tend to use PhpSpec usually and `shouldReturn` checks sameness.